task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +171,13 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    let x = Math.abs(circle.center.x - point.x);
+    let y = Math.abs(circle.center.y - point.y);
+    if (Math.hypot(x, y) < circle.radius) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
We can return (Math.hypot(x, y) < circle.radius)  with the same result
Please, see https://javascript.info/logical-operators